### ISSUE TITLE
Trying to import remote schema appears to remove local one

### ISSUE THREAD

**Issue by emanuelet** [almost-full-stack/graphcraft#28]
I'm trying to import the GraphQL schema from another micro-service that I have to create an association with a model.
When I try to set it up though in the main options, the change removes all the local sequelize models from the GraphQL schema.

```javascript
var options = {
  remote: {
    import: {
      RemoteData: {
        endpoint: 'http://localhost:3000/graphql',
        queries: { cardPaymentGet: { as: 'CardPaymentQuery' } },
      },
    },
  },
}

const { generateSchema } = require('sequelize-graphql-schema')(options)
```
Am I doing something wrong?

-----

![Screenshot_2019-09-06 Playground - http manuzbook 4000 graphql](https://user-images.githubusercontent.com/8566422/64413805-634e0f80-d0bc-11e9-9ca1-006bdfe0dc5a.png)
![Screenshot_2019-09-06 Playground - http manuzbook 4000 graphql(1)](https://user-images.githubusercontent.com/8566422/64413807-634e0f80-d0bc-11e9-8f55-7312ffeada60.png)

**Comment by alirizwan** [comment]
Hi, the remote schema functionality is not stable when it comes to importing a schema which is not based on sequelize-graphql-schema library. I must mention this in documentation (will update documentation). This part needs some refinements. I can always use some help on making this portion stable and would be happy if you want to contribute. Second thing this portion needs dataloader to be implemented. I wouldn't recommend using it for custom gql endpoints. being said that it should still not remove host schema like that (I will take a look at it and fix this in next release).

**Comment by emanuelet** [comment]
The weird part is that the other microservice is using `sequelize-graphql-schema` to generate its schema.

**Comment by alirizwan** [comment]
One image the bottom one clearly shows its using the seq-gql-schema but the other one (top image) don't have those pointers, there's no default query, the naming is off so I am not sure if the other endpoint is using the same lib or not. maybe you want to share some of your code (maybe privately) we can look at?

**Comment by emanuelet** [comment]
> One image the bottom one clearly shows its using the seq-gql-schema but the other one (top image) don't have those pointers, there's no default query, the naming is off so I am not sure if the other endpoint is using the same lib or not. maybe you want to share some of your code (maybe privately) we can look at?

So those are 2 screenshot of the same schema on the same server before and after I apply the `remote` configuration. The endpoints that are still there are custom ones that I've added manually

**Comment by alirizwan** [comment]
a better implementation for remote schema is in progress, hopefully it will be included in version `1.0.0`.
